code/modules/shuttle/shuttle.dm

```diff
@@ -552,6 +552,22 @@
 	else
 		. = null
 
+/obj/docking_port/mobile/proc/check_exile_pass()
+	if (is_station_level(z))
+		return TRUE
+	
+	for(var/mob/living/L in GLOB.alive_mob_list)
+		if(!istype(L))
+			continue
```

Comment:
This seems like incorrect approach. You should run it on mobs in shuttle's area instead of literally every mob in world.

Reply:
I tried and it would exclude mobs inside of containers.

Reply:
Mobs in the world is faster than stuff in the areas since you have to check contents

Reply:
It's also impossible for `L` to not be living as it's typed in the list search